src/main/java/com/github/dockerjava/core/util/CertificateUtils.java

```diff
@@ -41,115 +49,157 @@ public static boolean verifyCertificatesExist(String dockerCertPath) {
         return result;
     }
 
+    /**
+     * @param dockerCertPath with standard named files.
+     */
     public static KeyStore createKeyStore(final String dockerCertPath) throws NoSuchAlgorithmException,
             InvalidKeySpecException, IOException, CertificateException, KeyStoreException {
-        KeyPair keyPair = loadPrivateKey(dockerCertPath);
-        List<Certificate> privateCertificates = loadCertificates(dockerCertPath);
+        return createKeyStore("key.pem", "cert.pem");
```

Comment:
Tried to load into strings but I ended with a `javax.ws.rs.ProcessingException: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: signature check failed 
`